src/GenresModel.php

```diff
@@ -25,4 +25,27 @@ public function getAllGenres(): array
 
         return $genres;
     }
+
+    //get genre by genreID
+    public function getGenreByID(int $id)
+    {
+        $query = $this->db->prepare("SELECT* FROM `genre` WHERE `genre`.`id` = :genreID");
+
+        $query->bindParam('genreID', $id);
+
+        $query->execute();
+
+        $genreData = $query->fetch();
+
+        if (!$id) {
+            return false;
+        }
```

Comment:
This check should be the first thing in the function, failing as early as possible is always good